locale.html

```diff
@@ -90,6 +90,55 @@ <h1><ins>TimeZonesOfLocale ( _loc_ )</ins></h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-direction-of-locale" aoid=DirectionOfLocale>
+      <h1><ins>DirectionOfLocale ( _loc_ )</ins></h1>
+
+      <emu-alg>
+        1. <ins>If the default general ordering of characters (characterOrder) within a line in the _loc_ is right-to-left, then return *"rtl"*.</ins>
```

Comment:
No "the" before the variable name and no "then" in one-liners:
```suggestion
        1. <ins>If the default general ordering of characters (characterOrder) within a line in _locale_ is right-to-left, return *"rtl"*.</ins>
```